src/pieces/report.ts

```diff
@@ -20,9 +20,15 @@ async function register(bot: Client): Promise<void> {
 
 async function handleCron(bot: Client): Promise<void> {
 	const users: Array<SageUser> = await bot.mongo.collection(DB.USERS).find().toArray();
-	let report = 'Email,Count\n';
+	const courses: Array<string> = (await bot.mongo.collection(DB.COURSES).find().toArray()).map(course => course.name);
+
+	let report = `Email,Count,${courses.join(',')}\n`;
 	users.forEach(user => {
-		report += `${user.email},${user.count}\n`;
+		report += `${user.email},${user.count}`;
+		courses.forEach(course => {
```

Comment:
Did we say we wanted to only include courses that a student is enrolled in, not the ones that they're TAing for?

Reply:
`user.courses` only contains courses a student is enrolled in, not staffed ones. Ren's current document shows this.

Reply:
oop, my bad